plugins/Toolbox/src/CloudApiModel.py

```diff
@@ -0,0 +1,20 @@
+from typing import Union
```

Comment:
Why is this file not in the CloudSync folder?
Is it somehow different than the others? Serves a different purpose?

Reply:
Because it is used by Toolbox.py too, ie other scope than only Sync.